### ISSUE TITLE
ngrok support

### ISSUE THREAD

**Issue by W-i-n-7** [rejetto/hfs#462]
**Is your feature request related to a problem? Please describe.**no
primarily it would be easier for more novus users to have their site accessible anywhere if they cannot portforward like most home users cant
**Describe the solution you'd like**
hfs would ask for auth key and domain (both free)
then a start and stop ngrok button

**Comment by rejetto** [comment]
so, people would still need to make an account on ngrok, right?

**Comment by W-i-n-7** [comment]
> so, people would still need to make an account on ngrok, right?

yes but we can make a short guide to do it isnt too hard to get the auth key and make domain
creating account is the same as every other site

**Comment by BussyBakks** [comment]
maybe u can use hfs in this console and ngrok in that console (sorry for bad english)

**Comment by W-i-n-7** [comment]
ngrok has become horrible
its not worth it to add at all
not even if you have the enterprise plan ($39 for 15GB a month)
